<?xml version="1.0" standalone="yes"?>
<NewDataSet>
  <xs:schema id="NewDataSet" xmlns="" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
    <xs:element name="NewDataSet" msdata:IsDataSet="true" msdata:MainDataTable="Table1" msdata:UseCurrentLocale="true">
      <xs:complexType>
        <xs:choice minOccurs="0" maxOccurs="unbounded">
          <xs:element name="Table1">
            <xs:complexType>
              <xs:sequence>
                <xs:element name="Error_Type" type="xs:string" minOccurs="0" />
                <xs:element name="Error_DateTime" type="xs:dateTime" minOccurs="0" />
                <xs:element name="Error_Description" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Source" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Stack_Trace" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Caused_By_Method" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_1" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_2" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_3" type="xs:string" minOccurs="0" />
                <xs:element name="NombreUsuario" type="xs:string" minOccurs="0" />
                <xs:element name="HostName" type="xs:string" minOccurs="0" />
                <xs:element name="IPAddress" type="xs:string" minOccurs="0" />
              </xs:sequence>
            </xs:complexType>
          </xs:element>
        </xs:choice>
      </xs:complexType>
    </xs:element>
  </xs:schema>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2015-05-15T16:23:42.574806-05:00</Error_DateTime>
    <Error_Description>Índice fuera de los límites de la matriz.</Error_Description>
    <Error_Source>AplicacionSistemaVentura</Error_Source>
    <Error_Stack_Trace>   en AplicacionSistemaVentura.InterfazPrincipal.VerificarAlertas() en D:\Dropbox\Ventura\jrivas_20150506\AplicacionSistemaVentura\InterfazPrincipal.xaml.cs:línea 246</Error_Stack_Trace>
    <Error_Caused_By_Method>Void VerificarAlertas()</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>HADES</HostName>
    <IPAddress>192.168.1.2</IPAddress>
  </Table1>
</NewDataSet>